docs/processing-status-openapi-spec.yaml

```diff
@@ -587,23 +587,23 @@ paths:
           description: Bad request.  The upload identifier provided was not found.
         '401':
           description: Authorization information is missing or invalid.
-  '/upload/{destinationId}':
+  '/upload/{dataStreamId}':
     get:
       tags:
         - DEX Status
       summary: Provides all upload statuses for the given destination identifier, further filtered by the optional ext_event and date range parameters.
       description: >-
         Provides report for the given identifier.
       parameters:
-        - name: ext_event
+        - name: data_stream_route
           in: query
           schema:
             type: string
           description: >-
-            If specified, also filter by meta_ext_event that was provided with
+            If specified, also filter by meta_data_stream_route that was provided with
```

Comment:
This will be `data_stream_route` not `meta_data_stream_route`.

Reply:
I have corrected it.